constants/allowlistedTokens.ts

```diff
@@ -1,8 +1,8 @@
 //stores the metadata related to staticATokens to be deployed on l2
```

Comment:
naming is confusing, arrays have a similar name but contain different kind of information. one could rename it that way:
- `allowlistedStaticATokens` --> `allowlistedStaticATokensData`
- `allowlistedATokens` --> `allowlistedATokensAddresses`